octohatrack/__init__.py

```diff
@@ -17,13 +17,17 @@ def main():
 
   repo_name = args.repo_name
 
-  if not repo_exists(repo_name): 
-    print("Repo does not exist: %s" % repo_name)
+  try:
```

Comment:
The try/catch is to capture the API :boom: when the `ValueError` is thrown in [response.py](https://github.com/LABHR/octohatrack/blob/b1a56f6e91dc85e2d72c8fc8a56279503c31c4f4/octohatrack/response.py#L103)